.github/workflows/create_release.yml

```
@@ -37,7 +37,7 @@ jobs:
           base: main
           title: Release ${{ steps.version.outputs.next_tag }}
           labels: chore
-          reviewers: krivard
-          assignees: krivard
+          # reviewers: krivard
```

just a suggestion -- im fine with you leaving an easter egg ;)
```suggestion
          # reviewers:
```

---

lol